presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,436 @@ Except for the limit on queued queries, when a resource group runs out of a reso
 it does not cause running queries to fail; instead new queries become queued.
 A resource group may have sub-groups or may accept queries, but may not do both.
 
-The resource groups and associated selection rules are configured by a manager which is pluggable.
-Add an ``etc/resource-groups.properties`` file with the following contents to enable
-the built-in manager that reads a JSON config file:
+In PrestoDB, Resource Groups are a powerful tool for managing query execution and resource allocation.
+They allow administrators to control how resources are allocated and utilized on a Presto cluster.
 
-.. code-block:: none
+Resource Usage Limits
+---------------------
+
+Resource Groups can set limits on resource usage, such as CPU time, memory usage, or total
+number of queries. This is particularly useful in multi-tenant environments where you want to ensure that no single user
+or query monopolizes the system resources.
+
+Resource Consumption
+--------------------
+
+A query belongs to a single resource group, and it consumes resources from that group,
+as well as its parent groups. If a resource group runs out of a certain resource, it does not cause running
+queries to fail. Instead, new queries will be queued until resources become available again.
+
+Sub-Groups and Query Acceptance
+-------------------------------
+
+Sub-groups allow for hierarchical resource allocation, where each sub-group can have its own resource limits and
+queueing policies.A resource group that accepts queries directly is a leaf group, and it executes queries using
+its allocated resources.
+
+The resource groups and associated selection rules are configured by a manager, which is pluggable.
+Presto resource management can be done in two ways:
+
+* File-Based Resource Management
+* Database-Based Resource Management
```

Comment:
how about having the following `File-Based Resource Management` and `Database-Based Resource Management` sections as the sub-items here directly? Just change them to `^^^^^^^^^^^^^^^^^^^^^^^^^` and remove these two entries?